docs/source/user_guide.rst

```diff
@@ -1,5 +1,5 @@
-User guide
-==========
+Use poliastro
```

Comment:
I think we can keep this as "user guide", it's a very common term

Reply:
Sure, no problem with that!